src/app/app.module.ts

```diff
@@ -9,17 +9,44 @@ import { ApiConfigurationService } from './services/api-configuration.service';
 import { MovieDetailsModule } from './pages/movie-details/movie-details.module';
 import { HeaderModule } from './components/header/header.module';
 import { TvDetailsModule } from './pages/tv-details/tv-details.module';
+import { GuestSessionService } from './services/guest-session.service';
+import { RatingService } from './services/rating.service';
+import { forkJoin, of } from 'rxjs';
+import { NotificationModule } from './components/notifications/notifications.module';
+
+const ratingInitializerFactory =
```

Comment:
Maybe you can move this functions to /factory folder somewhere

Reply:
Thanks,  moved.